src/App.tsx

```diff
@@ -10,23 +10,45 @@ import Profile from "./pages/Profile/Profile";
 import Login from "./pages/Login/Login";
 import Register from "./pages/Register/Register";
 import Account from "./pages/Account/Account";
+import { doc, getDoc } from "firebase/firestore";
+import { Task } from "./mockData/mockActiveTasks";
 import { UserProfile } from "./mockData/mockTribe";
+import { useEffect, useState } from "react";
 import { app, db } from "./firebase";
 import { collection, getDocs, getFirestore, query } from "firebase/firestore";
-import { doc, getDoc } from "firebase/firestore";
-import { CompletedTask as CompletedTaskType } from "./mockData/mockCompletedTasks";
-import { useEffect, useState } from "react";
+import {
+  CompletedTask as CompletedTaskType,
+  completedTasks,
+} from "./mockData/mockCompletedTasks";
 import { Dayjs } from "dayjs";
 
 const App = () => {
-  const [userUID, setUserUID] = useState<null | string>(null)
+  const [userUID, setUserUID] = useState<null | string>(null);
   // NOTE: this console.log is used to workaround an eslint warning
   // It should be deleted once userUID is used
-  console.log(userUID)
+  console.log(userUID);
+
+  const [activeTasksList, setActiveTasksList] = useState<Task[]>([]);
+
+  useEffect(() => {
+    const getTasks = async () => {
+      const db = getFirestore(app);
```

Comment:
You can import the db from `firebase.ts` this means that we do not have to initialise it every time we run a request